runtime/include/chpl-file-utils.h

```diff
@@ -40,6 +40,46 @@ qioerr chpl_fs_cwd(const char** working_dir);
 
 qioerr chpl_fs_exists(int* ret, const char* name);
 
+/*
+  Get the UID of the calling process. This function is UNIX only and cannot
+  fail.
+*/
+int chpl_fs_get_cuid(void);
```

Comment:
Side note: I typically prefer documentation to be on the implementation rather than the declaration of functions.  This makes it more likely that the documentation will get updated if/when the function is changed.